README.md

```diff
@@ -45,9 +45,10 @@ These can be retrieved from the App Registration overview blade.
 
 The following roles need [setting up](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-add-app-roles-in-azure-ad-apps)
 
-- Payment.Holds.Admin
+- Payment.Hold.Admin
 - Payment.Scheme.Admin
 - Payment.Data.View
+- Payment.Closure.Admin
```

Comment:
These will need to be setup in Azure App Registration in all environments for this to work.

Reply:
(doesn't need to be done ahead of time, just means that scope will never exist for anyone and they won't be able to request this role)